### PR TITLE
[miniaudio] Fixing miniaudio and Makefile for NetBSD

### PR DESCRIPTION
**Pull request by NishiOwO**

### CODE REVIEW

File: src/Makefile [hunk 1]
```diff
@@ -467,7 +467,7 @@ INCLUDE_PATHS = -I.
 ifeq ($(TARGET_PLATFORM),PLATFORM_DESKTOP_GLFW)
     INCLUDE_PATHS += -Iexternal/glfw/include
     ifeq ($(PLATFORM_OS),BSD)
-        INCLUDE_PATHS += -I/usr/local/include
+        INCLUDE_PATHS += -I/usr/local/include -I/usr/pkg/include -I/usr/X11R7/include
```
Comment (raysan5) [inline src/Makefile:470]: Those paths look highly irregular, why are they needed? Is there no other option?

File: src/external/miniaudio.h [hunk 2]
```diff
@@ -36078,9 +36078,15 @@ static ma_result ma_context_get_device_info_from_fd__audio4(ma_context* pContext
         ma_uint32 channels;
         ma_uint32 sampleRate;
 
+#ifdef __NetBSD__
```
Comment (raysan5) [inline src/external/miniaudio.h:36081]: Actually those changes should be on miniaudio side because they will be ovewritten when updating miniaudio...

Comment (NishiOwO) [reply src/external/miniaudio.h:36081]: already merged into dev branch of miniaudio, so it is for until we get new release

File: src/Makefile (hunk 1 above)
Comment (NishiOwO) [reply src/Makefile:470]: because netbsd installs packages under /usr/pkg

**Comment by raysan5** [comment]
Merging! Note miniaudio will be override on next update!

**Pull request merged**